Source/WebCore/PAL/pal/spi/cocoa/NSAttributedStringSPI.h

```diff
@@ -97,7 +97,7 @@ SOFT_LINK_CONSTANT(UIFoundation, NSTextAlternativesAttributeName, NSString *)
 #define NSTextAlternativesAttributeName getNSTextAlternativesAttributeName()
 
 #if HAVE(NSTEXTLIST_MARKER_FORMATS)
-#if USE(APPLE_INTERNAL_SDK)
+#if __has_include(<UIKit/NSTextList.h>)
```

Comment:
One thing that I noticed while quickly scrolling through this patch: it's better to use OS version macros when possible, to make it obvious when obsolete code can be removed. An unnecessary `__has_include` branch can stay for years, as it's hard to find and hard to confirm that it's unneeded.